be/src/service/backend_service.cpp

```diff
@@ -705,8 +705,8 @@ void BackendService::ingest_binlog(TIngestBinlogResult& result,
 
     if (local_tablet->enable_unique_key_merge_on_write()) {
         StorageEngine::instance()->txn_manager()->set_txn_related_delete_bitmap(
-                partition_id, txn_id, local_tablet_id, local_tablet->schema_hash(),
-                local_tablet->tablet_uid(), true, delete_bitmap, pre_rowset_ids);
```

Comment:
Now need test P0 case, so add this.
the fix pr is https://github.com/apache/doris/pull/23789